### ISSUE TITLE
Eigen version confirmation

### ISSUE THREAD

**Issue by Shar-pei-bear** [tiev-tongji/quadric_slam#3]
Hi,

I am having some trouble compiling this project due  to eigen version conflict. Could you please confirm which eigen version do you use to compile this project? Thank you for your help.

Best,
Zhentian Qian

**Comment by JerryAIwei** [comment]
Hi,
Sorry to respond so late. My eigen version is 3.3.4. If you want to test the quadric slam, please run `git reset --hard ca08b0a1512b35fd3e90bfebe1bd27cf7278097b `. There are bugs in the current commit, I will fix them in two weeks.

**Comment by rantengsky** [comment]
> Hi,
> 
> I am having some trouble compiling this project due to eigen version conflict. Could you please confirm which eigen version do you use to compile this project? Thank you for your help.
> 
> Best,
> Zhentian Qian

have you solved the eigen problem?

**Comment by Shar-pei-bear** [comment]
> > Hi,
> > I am having some trouble compiling this project due to eigen version conflict. Could you please confirm which eigen version do you use to compile this project? Thank you for your help.
> > Best,
> > Zhentian Qian
> 
> have you solved the eigen problem?
Yes, by following the author's instruction, I am able  to compile and run the program.

**Comment by rantengsky** [comment]
but the version ca08b0a that recommended have no data association, right?
